### DISCUSSION TITLE
Upgrading Docker Install 9.7.0 to 9.9.0

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#12903] [Archive]
Hi,

I was trying to upgrade by just changing the version number in my docker compose file. When I bring the server up, it error out with this message:

```
cms_1   | 21:19:46 ✨ Initializing bootstrap...
cms_1   | 21:19:46 🚨 Knex: run
cms_1   | $ npm install sqlite3 --save
cms_1   | Error relocating /directus/node_modules/sqlite3/lib/binding/napi-v6-linux-x64/node_sqlite3.node: fcntl64: symbol not found
cms_1   | Error: Error relocating /directus/node_modules/sqlite3/lib/binding/napi-v6-linux-x64/node_sqlite3.node: fcntl64: symbol not found
cms_1   |     at Object.Module._extensions..node (node:internal/modules/cjs/loader:1187:18)
cms_1   |     at Module.load (node:internal/modules/cjs/loader:981:32)
cms_1   |     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
cms_1   |     at Module.require (node:internal/modules/cjs/loader:1005:19)
cms_1   |     at require (node:internal/modules/cjs/helpers:102:18)
cms_1   |     at Object.<anonymous> (/directus/node_modules/sqlite3/lib/sqlite3-binding.js:4:15)
cms_1   |     at Module._compile (node:internal/modules/cjs/loader:1103:14)
cms_1   |     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
cms_1   |     at Module.load (node:internal/modules/cjs/loader:981:32)
cms_1   |     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
```

Is this a sqllite specific problem?

I'm trying to migrate off to an instance running 9.9.0 on MySQL. Some of the data exports/imports didn't work and I'm assuming it is because I need to make the old one run on 9.9.0 first.

thanks
  Matt

**Comment by azrikahar** [comment]
I can confirm the exact same error on my end (Docker image specifically). Seems to be a known issue TryGhost/node-sqlite3#1576 with `sqlite3@5.0.3`. It's technically resolved with the release of `5.0.4` 3 days ago, but I believe since the docker image for latest/9.9.0 tags was built 5-6 days ago, it is unfortunately still using `5.0.3`. 

Did a quick `npm list sqlite3` before the container eventually fails:

![WindowsTerminal_clasmcE8Ln](https://user-images.githubusercontent.com/42867097/164392573-0f878029-049c-4bea-98a2-487c2f766186.png)

cc @rijkvanzanten

**Reply by rijkvanzanten** [reply]
ugh that's annoying. at least glad to hear the problem is already resolved and will be included in the next release

**Comment by Joey92** [comment]
I'm getting it, too. I'm trying to run it on a read-only filesystem though. Might that be the issue?
